### ISSUE TITLE
Please support TFLite gradient operations: BroadcastGradientArgs, StridedSliceGrad and some other

### ISSUE THREAD

**Issue by barabanus** [tensorflow/tensorflow#50595]
Within our company we trained a breakthrough realtime model which requires automatic differentiation feature at runtime. To be more specific, the model's output should be forward pass outputs + derivative of runtime loss function with respect to some input control variables.

We use TFLite to run the model, but we can't use TF Select kernels because our runtime environment is very strict (video gaming consoles). In the current state, when we try to use `tf.gradients` within our model the TFLite converter does not convert because of some operations to be not supported: `BroadcastGradientArgs`, `DynamicStitch`, `EluGrad`, `Sign`, `StridedSliceGrad`, `UnsortedSegmentSum`

As a workaround, we approximate the derivatives with finite difference method, but obviously it requires to make extra forward passes + it introduces some numerical instability to the model. It would be very helpful to have cheap analytical derivatives with a backpropagation step within TFLite.

**System information**
- `Linux rb15 5.12.14-arch1-1 #1 SMP PREEMPT Thu, 01 Jul 2021 07:26:06 +0000 x86_64 GNU/Linux`
- `TensorFlow 2.5.0` from `tensorflow-opt-cuda` package (https://archlinux.org/packages/community/x86_64/tensorflow-opt-cuda)

**Provide the text output from tflite_convert**
```
Some ops are not supported by the native TFLite runtime, you can enable TF kernels fallback using TF Select. See instructions: https://www.tensorflow.org/lite/guide/ops_select 
TF Select ops: BroadcastGradientArgs, DynamicStitch, EluGrad, Sign, StridedSliceGrad, UnsortedSegmentSum
Details:
	tf.BroadcastGradientArgs {device = ""}
	tf.DynamicStitch {device = ""}
	tf.EluGrad {device = ""}
	tf.Sign {device = ""}
	tf.StridedSliceGrad {begin_mask = 0 : i64, device = "", ellipsis_mask = 0 : i64, end_mask = 0 : i64, new_axis_mask = 0 : i64, shrink_axis_mask = 0 : i64}
	tf.StridedSliceGrad {begin_mask = 0 : i64, device = "", ellipsis_mask = 0 : i64, end_mask = 0 : i64, new_axis_mask = 0 : i64, shrink_axis_mask = 1 : i64}
	tf.StridedSliceGrad {begin_mask = 0 : i64, device = "", ellipsis_mask = 0 : i64, end_mask = 1 : i64, new_axis_mask = 0 : i64, shrink_axis_mask = 0 : i64}
	tf.StridedSliceGrad {begin_mask = 0 : i64, device = "", ellipsis_mask = 1 : i64, end_mask = 0 : i64, new_axis_mask = 2 : i64, shrink_axis_mask = 0 : i64}
	tf.StridedSliceGrad {begin_mask = 1 : i64, device = "", ellipsis_mask = 0 : i64, end_mask = 0 : i64, new_axis_mask = 0 : i64, shrink_axis_mask = 0 : i64}
	tf.StridedSliceGrad {begin_mask = 1 : i64, device = "", ellipsis_mask = 0 : i64, end_mask = 1 : i64, new_axis_mask = 0 : i64, shrink_axis_mask = 0 : i64}
	tf.StridedSliceGrad {begin_mask = 1 : i64, device = "", ellipsis_mask = 0 : i64, end_mask = 1 : i64, new_axis_mask = 0 : i64, shrink_axis_mask = 2 : i64}
	tf.StridedSliceGrad {begin_mask = 1 : i64, device = "", ellipsis_mask = 0 : i64, end_mask = 1 : i64, new_axis_mask = 2 : i64, shrink_axis_mask = 0 : i64}
	tf.StridedSliceGrad {begin_mask = 3 : i64, device = "", ellipsis_mask = 0 : i64, end_mask = 3 : i64, new_axis_mask = 0 : i64, shrink_axis_mask = 0 : i64}
	tf.StridedSliceGrad {begin_mask = 3 : i64, device = "", ellipsis_mask = 0 : i64, end_mask = 3 : i64, new_axis_mask = 0 : i64, shrink_axis_mask = 4 : i64}
	tf.UnsortedSegmentSum {device = ""}
```

**Comment by thaink** [comment]
It will take time to review and add ops. You can use https://www.tensorflow.org/lite/guide/ops_select at the moment to unblock the conversion.

**Comment by avroshk** [comment]
@thaink is there any thread or doc where we can see which ops are under review for the upcoming tensorflow releases? It will help us immensely to plan our internal projects that need to run in really constrained tflite runtime environments and cannot use select ops. Thanks!

**Comment by thaink** [comment]
The requests are tracked in https://github.com/tensorflow/tensorflow/issues/21526

**Comment by pkgoogle** [comment]
Hi @barabanus, it seems like the ops are now supported according to  https://github.com/tensorflow/tensorflow/issues/21526, please close as completed if there are no other open items regarding this. Thanks.

**Comment by barabanus** [comment]
Thanks, @pkgoogle
